packages/wrap/src/transforms/MapLeafValues.ts

```diff
@@ -90,7 +90,7 @@ export default class MapLeafValues implements Transform<MapLeafValuesTransformat
     transformationContext: MapLeafValuesTransformationContext
   ): ExecutionRequest {
     const document = originalRequest.document;
-    const variableValues = (originalRequest.variables = {});
+    const variableValues = originalRequest.variables || {};
```

Comment:
Thank you so much, I guess ?? could be a tad more proper than ||, but mileage may vary. Otherwise looks great to me

Reply:
Agree. PR updated. Changed `||` for `??`. Thanks.